charts/core/src/Axis/Axis.ts

```diff
@@ -0,0 +1,63 @@
+import { CartesianAxisOption } from 'echarts/types/src/coord/cartesian/AxisModel.js';
+
+import { Theme } from '@leafygreen-ui/lib';
+import {
+  color,
+  fontFamilies,
+  fontWeights,
+  InteractionState,
+  spacing,
+  Variant,
+} from '@leafygreen-ui/tokens';
+
+import { type AxisProps } from './Axis.types';
+
+export const getAxisOptions = (
+  theme: Theme,
+  props: AxisProps,
+): CartesianAxisOption => {
+  const { label, type } = props;
+
+  return {
+    type,
+    ...(type !== 'category' && { min: props.min }),
+    ...(type !== 'category' && { max: props.max }),
+    ...(type === 'category' && { data: props.data }),
```

Comment:
I think I'd recommend something like this:

```suggestion
  const { label, type } = props;
  const isContinuous = type !== 'category'; 

  return {
    type,
    ...(isContinuous ? { data: props.data } : { min: props.min, max: props.max }),
```

Reply:
Wouldn't it be:
```
    ...(isContinuous ? { min: props.min, max: props.max } : { data: props.data })
```

Reply:
Your suggestion is cleaner, I like it! Applied it in the commit.